scripts/sm-dev

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+# Inspired by https://github.com/grafana/synthetic-monitoring-api/tree/main/scripts#sm-dev
```

Comment:
while correct, most people won't have access to this.

Reply:
Erased!